RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+:exc:`~hypothesis.errors.HypothesisDeprecationWarning` now inherits from
+:exc:`python:FutureWarning` instead of :exc:`python:DeprecationWarning`,
+as recommended by :pep:`565` for user-facing warnings (:issue:`618`).
```

Comment:
I think it would be worth being more explicit about the consequences for this - e.g. that the warnings will not be hidden by default.

Reply:
Done.